frontend/src/Components/Dashboard/index.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import { Layout, Menu } from "antd";
+import {
+  UserOutlined,
+  LaptopOutlined,
+  NotificationOutlined,
+} from "@ant-design/icons";
+import "./index.css";
+
+import Profile from "../Profile";
+import Team from "../Team";
+import NotFound from "../NotFound";
+import { getUserID } from "../Firebase_funcs/firebase_funcs";
+
+const Dashboard = () => {
+  const { Content, Sider } = Layout;
```

Comment:
This can (and should) be outside the component. It doesn't depend on any data from this component and having this inside it would make this variable to be recreated each time a state / props (basically any lifecycle method being called) changes.